### PR TITLE
HackIllinois 2018 Recap

### PR DESCRIPTION
**Pull request by justinyangusa**
Started a blog post to recap HackIllinois 2018. Right now, I summarized some of the events that took place, but I'd like to collect some thoughts on what direction we want to take this piece before going further.

Here are some things I plan on adding to the post:
- More examples/details about the open-source projects/mentors
- Stacktistics about the event
- Images/media

### CODE REVIEW

**Comment by justinyangusa** [comment]
+label: WIP

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2018-04-28 00:00:00
+categories: recap
+tags: HackIllinois2018
+image: /assets/article_images/hackillinois_2018.jpg
+author: Justin Yang
+---
+
+This February, HackIllinois welcomed over a thousand students to the University of Illinois at Urbana-Champaign for an excitement-packed weekend to develop open-source projects.
+
+HackIllinois 2018 kicked off Friday afternoon, with students lining up for registration and to chat with sponsors at the company expo. After everyone filled up on dinner from Panda Express and Kohinoor, they heard opening remarks from the Opening Ceremony. Attendees then broke out into two panels based on their interests: Web & Data and Systems where open-source mentors spoke about their projects and the open-source community in general.
```
Comment (smg155) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: This is picky, but I would do away with the comma and consider rephrasing the second part of the sentence

Comment (smg155) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: I mean for the first sentence in this paragraph.

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Attendees, however, weren't just at HackIllinois to hear about open source. As the panels concluded, attendees formed into groups via the project matching sessions to work on open-source projects -- be it with open-source projects to improve popular open-source projects such as Rust and Freenode.
```
Comment (smg155) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: Reduce the amount of times saying "open-source" in this paragraph. Also the "be it" part sounds a little confusing grammatically.

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Groups worked on their projects throughout the weekend with the goal of starting meaningful contributions to their open-source projects. As attendees worked with their mentors to learn the ups and downs of their projects and strode to make changes and submit pull requests, along the way they learned about new technologies at workshops ranging from virtual reality to machine learning, as well as lightning talks from fellow students about such disparate topics as memes and documentation.
+
+HackIllinois 2018 also featured keynotes from Katy Huff, Illinois professor in NPRE; Steve Klabnik, Rust; Vijay Pai, Google; and Travis Oliphant, numpy/scipy/anaconda; who spoke about powerful ideas in the open-source community.
+
+Still, it wasn't all about pure development. Sppnsors provided plenty of fun activities to take a break, including video games and puzzles in their lounges, for instance. Many attendees also stepped out for fun games of nerf or to play with the therapy dogs.
```
Comment (smg155) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: "Sponsors"

**Comment by justinyangusa** [comment]
Thanks @smg155! Reworked most parts of the post—check it out.

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 4]
```diff
@@ -1,28 +1,47 @@
...
+
+Throughout the weekend, there were all sorts of activities to take a break from coding. Our sponsors set out swag at their tables and games in their lounges.
+
+![Pickles the wolf](/assets/article_images/pickles.jpg "Pickles the wolf hangs out in the IMO lounge")
+
+Tons of attendees had fun playing Nerf at the Kenney Gym on Saturday. As is tradition, the last game was a Hackillinois staff versus sponsor round.
+
+![Nerf battle](/assets/article_images/nerf.jpg "Battling it out at Nerf")
 
+Sunday morning brought the end of hacking, and groups wrapped up their contributions and documented their work on Devpost in preparation for demos. Some therapy dogs visited and brought a smile to many a face.
```
Comment (apa-123) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: is "brought a smile to many a face" a phrase? would probably make more sense with "brought smiles to many faces"

Comment (apa-123) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: is "brought a smile to many a face" a phrase? would probably make more sense with "brought smiles to many faces"

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 5]
```diff
@@ -1,28 +1,47 @@
...
+![opening ceremony](/assets/article_images/ceremony.jpg "HackIllinois 2018 kicks off!")
 
-Attendees, however, weren't just at HackIllinois to hear about open source. As the panels concluded, attendees formed into groups via the project matching sessions to work on open-source projects -- be it with open-source projects to improve popular open-source projects such as Rust and Freenode.
+With the focus on open source at HackIllinois, attendees spent the weekend learning about and building out new open source features and tools that tackled issues in many different fields. HackIllinois kicked off with panels on Web/Data and Systems/Miscellaneous after the Opening Ceremony. Mentors presented some of the projects students could join, after which students joined groups they were interested in.
```
Comment (apa-123) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: "Mentors presented their projects, after which students joined groups they were interested in."

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 6]
```diff
@@ -1,28 +1,47 @@
...
+date: 2018-05-19 00:00:00
 categories: recap
 tags: HackIllinois2018
-image: /assets/article_images/hackillinois_2018.jpg
+image: /assets/article_images/2018_recap_cover.jpg
 author: Justin Yang
 ---
 
-This February, HackIllinois welcomed over a thousand students to the University of Illinois at Urbana-Champaign for an excitement-packed weekend to develop open-source projects.
+This February, HackIllinois welcomed over 1000 students to the University of Illinois at Urbana-Champaign for a developement-filled weekend where students from across the Midwest and the country gathered to make contributions to open source projects. Over the course of just 36 hours, attendees made signifigant contributions, making over [94 submissions](https://hackillinois-2018.devpost.com/submissions) by the end!
```
Comment (apa-123) [inline _posts/2018-04-28-hackillinois-2018-recap.markdown]: significant** contributions

File: _posts/2018-04-28-hackillinois-2018-recap.markdown (hunk 4 above)
Comment (justinyangusa) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: I believe the phrase is idiomatic, but I’ll change it.

File: _posts/2018-04-28-hackillinois-2018-recap.markdown (hunk 5 above)
Comment (justinyangusa) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: 👍

File: _posts/2018-04-28-hackillinois-2018-recap.markdown (hunk 6 above)
Comment (justinyangusa) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: Ok

File: _posts/2018-04-28-hackillinois-2018-recap.markdown [hunk 7]
```diff
@@ -1,28 +1,47 @@
...
+![panel](/assets/article_images/panel.jpg "Mentors talk about open source at the Web/Data panel")
 
-HackIllinois 2018 also featured keynotes from Katy Huff, Illinois professor in NPRE; Steve Klabnik, Rust; Vijay Pai, Google; and Travis Oliphant, numpy/scipy/anaconda; who spoke about powerful ideas in the open-source community.
+Attendees worked late into the night starting on their projects: mentors taught students how to set up their environments and basics of the project and how to make contributions.
 
-Still, it wasn't all about pure development. Sppnsors provided plenty of fun activities to take a break, including video games and puzzles in their lounges, for instance. Many attendees also stepped out for fun games of nerf or to play with the therapy dogs.
+![food](/assets/article_images/food.jpg "Attendees fill up on food to fuel them through the night")
```
[the remark this answers is not in the document]
Comment (justinyangusa) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: Agreed, I think that’s a good idea. I’d like for the open source article to touch upon sponsors in their capacity as mentors as well.

As an aside, we should make sure sponsors (and others) are aware of the blog. As it stands right now, I don’t believe there are many (or any) links back to the blog on the main site or our social media. We should put some links there, and also in our prospectus or sponsor emails.

[the remark this answers is not in the document]
Comment (rohinb2) [reply _posts/2018-04-28-hackillinois-2018-recap.markdown]: Yeah, I usually link blog.hackillinois.org in the emails. Adding it to social media is also a good idea and @apa-123 and @smg155 can do that if they agree.

**Comment by justinyangusa** [comment]
Markdown images seem to be showing the alt text instead of the title text in production. I’m afk at the moment, but if someone wants to swap those fields in the meantime, feel free.

**Pull request merged**